### PR TITLE
DS-679 | @ericbakenhus | Stub synchronized event component

### PR DESCRIPTION
**Pull request by ericbakenhus**
# READY FOR REVIEW

# Summary
- Stub synchronized event component (so _something_ shows up in the editing UI)

# Review By (Date)
- End of sprint

# Criticality
- Low

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at code
2. In Storyblok (DEV), check out a test event: `/events/sync/test`
3. In Storyblok (DEV), check out the Synchronized Event content type

# Associated Issues and/or People
- DS-679

### CODE REVIEW

File: src/components/content-types/synchronized-event/synchronizedEvent.js
Comment (ericbakenhus) [inline src/components/content-types/synchronized-event/synchronizedEvent.js:1]: This is literally just a slightly tweak copy of the existing Event component. Meant as a stub.

File: src/components/content-types/synchronized-event/synchronizedEvent.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+import { Heading } from '../../simple/Heading';
+import { FlexBox } from '../../layout/FlexBox';
+import { SrOnlyText } from '../../accessibility/SrOnlyText';
+import SbLink from '../../../utilities/sbLink';
+import CardImage from '../../media/cardImage';
+import TabLabel from '../../simple/tabLabel';
+import DateBlock from '../../simple/dateBlock';
+import HeroIcon from '../../simple/heroIcon';
+
+// THIS IS A STUB; COMPLETE THE OWL PLZ
```
Comment (ericbakenhus) [inline src/components/content-types/synchronized-event/synchronizedEvent.js:15]: If anyone knows the ticket for building out the event UI, please feel free to edit this TODO. 😅

**Comment by stfurkan** [comment]
@sherakama @ericbakenhus I think we should use a branch for all the `events discovery` changes instead of merging them to `dev` branch. I created the **[events-discovery](https://github.com/SU-SWS/saa_alumni/tree/events-discovery)** branch . What do you think?

**Comment by ericbakenhus** [comment]
> @sherakama @ericbakenhus I think we should use a branch for all the `events discovery` changes instead of merging them to `dev` branch. I created the **[events-discovery](https://github.com/SU-SWS/saa_alumni/tree/events-discovery)** branch . What do you think?

Sounds good to me. 👍

**Comment by sherakama** [comment]
Makes sense to me. Thanks.

**Review by stfurkan — APPROVED** [review]
Thanks @ericbakenhus :)

Should the component's `Custom Data (Overrides)` section work in Storyblok? It's not working currently but since this is just a stub, I'm assuming that's fine for now :D 

image

**Comment by ericbakenhus** [comment]
> Thanks @ericbakenhus :)
> 
> Should the component's `Custom Data (Overrides)` section work in Storyblok? It's not working currently but since this is just a stub, I'm assuming that's fine for now :D

No, I'm not expecting it to work. This ticket is just to create the fields. I tossed in the stub so we have a place to drop a copy of the new event UI once it's built. I don't see an individual ticket for that, but I'm guessing it's part of DS-682. 🤔 

I'll create a more obvious follow up ticket to handle this preview component work, so it doesn't get lost. Thanks!

**Pull request merged**